litejpeg/core/huffman/huffmancore.py

```diff
@@ -0,0 +1,479 @@
+"""
+HuffmanEncoder Module:
+----------------------
+The module takes an input in the form of matrix containing 64 blocks with
+20 bits each and give an output of variable number each having a bit_length
+of 8 bits.
+The module is connected to the other modules containing Huffman Tables which
+are required for the process of encoding the input data.
+This module takes input from the RLE module containing three paramenters:
```

Comment:
Check spelling...

You could improve the formatting of this comment like follows;

```
This module takes input from the RLE module containing three parameters:
    1. amplitude: Non-zero input in the input matrix of the RLE.
    2. run length: Number of zeros before the non-zero amplitude.
    3. size: Number of bits required to store amplitude.

Each of the output is of 8 bits.

The main purpose of the module is to serialize the data and encode the data
using Huffman tables.

The advantages of the encoding scheme is that boundaries between characters
are easily determined, and the pattern used for each character is fixed and universal.
```

Reply:
Done